test_files/reexport.declaration/clutz_user.d.ts

```diff
@@ -0,0 +1,9 @@
+/** @fileoverview Tests that renamed exports get emitted with the correct clutz alias names. */
```

Comment:
This change adds a d.ts but has no corresponding code or baseline change ... ?

Is it just testing that this code compiles?

Reply:
Yes. We have a test (e2e_dts_test or so) that just passes all of these through tsc.